lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -279,15 +278,6 @@ MStatus MayaUsdProxyShapeBase::initialize()
     retValue = addAttribute(excludePrimPathsAttr);
     CHECK_MSTATUS_AND_RETURN_IT(retValue);
 
-    loadPayloadsAttr
-        = numericAttrFn.create("loadPayloads", "lpl", MFnNumericData::kBoolean, 1.0, &retValue);
-    CHECK_MSTATUS_AND_RETURN_IT(retValue);
-    numericAttrFn.setKeyable(true);
-    numericAttrFn.setReadable(false);
-    numericAttrFn.setAffectsAppearance(true);
-    retValue = addAttribute(loadPayloadsAttr);
-    CHECK_MSTATUS_AND_RETURN_IT(retValue);
-
```

Comment:
In this file, we're removing the load-payloads proxy shape attribute.

Reply:
What happens with a saved Maya scene where this attribute was set and saved in the Maya file?

Reply:
Maya does not allow removing attribute? I can put it back in the node definition but make it do nothing instead. Is that how legacy attribute must be handled?